concent_api/core/tasks.py

```diff
@@ -125,7 +125,7 @@ def verification_result(
 
     result_enum = VerificationResult[result]
 
-    assert result_enum != VerificationResult.ERROR or all([error_message, error_code])
+    assert result_enum != VerificationResult.ERROR or all([isinstance(error_message, str), isinstance(error_code, str)])
```

Comment:
Fix looks good, but I think we should also determine on what occasion there is an empty `error_message` or `error_code` and fix that too. From what I see we always set `error_message` or `error_code` when raising `VerificationError` so this should not happen.

Btw. linked sentry issue doesn't look related to this case.

Reply:
After discussion, looks like the cause is already fixed.